C++/Logger/sinks/Sink.cpp

```diff
@@ -0,0 +1,12 @@
+#include "Sink.h"
+
+namespace Toolbox::Log::Sinks
+{
+
+Sink & Sink::operator << (const std::string & message)
+{
+  this->output (message);
+  return *this;
```

Comment:
I believe it is good to have the * and & next to the object when it is a dereference and address-of operator.  I agree with the space when it is a pointer declaration.